pyproject.toml

```
@@ -0,0 +1,88 @@
+[build-system]
+requires = [
+    "setuptools>=65",
+    "wheel",
+    # leave Cython as an optional, runtime‑selectable extra
+    # setuptools will still import it from setup.py when requested
+    "cython",
+    "pybind11",
+]
+build-backend = "setuptools.build_meta"
+
+# project metadata
+
+[project]
+name = "pyomo"
+description = "The Pyomo optimization modeling framework"
+readme = "README.md"
+license = { file = "LICENSE.md" }
+authors = [{name = "Pyomo Development Team", email = "pyomo-developers@googlegroups.com"}]
```

```suggestion
authors = [{name = "Pyomo Development Team", email = "wg-pyomo@sandia.gov"}]
```
@jsiirola @mrmundt should we update the email address to match what we have in the README to request dev call info?

---

@jsiirola @blnicho - I say "yes". But if so, we should check anywhere else that that one is used (e.g., `RELEASE.md`)

---

My only concern is that only the SNL developers receive mail sent to `wg-pyomo`.  While that does hit the bulk of the current "core dev team", it does not get to everyone (and I personally don't want to give people the untrue impression that you have to be at SNL to be a developer).  `pyomo-developers` is more public and can reach anyone who has developer interest.  

The justification for `wg-pyomo` for the dev call info is that the meeting is hosted by SNL, so it makes sense to request the info from an SNL address.

---

I accept that counter point - optics are important!